third_party/2and3/werkzeug/datastructures.pyi

```diff
@@ -402,24 +402,34 @@ class WWWAuthenticate(UpdateDictMixin, dict):
     qop: Any
     stale: Any
 
-class FileStorage:
-    name: Any
-    stream: Any
-    filename: Any
-    headers: Any
-    def __init__(self, stream: Optional[Any] = ..., filename: Optional[Any] = ..., name: Optional[Any] = ...,
-                 content_type: Optional[Any] = ..., content_length: Optional[Any] = ..., headers: Optional[Any] = ...): ...
+class _Writer(Protocol):
+    def write(self, data: bytes) -> Any: ...
+
+class FileStorage(object):
+    name: Optional[Text]
+    stream: IO[bytes]
+    filename: Optional[Text]
+    headers: Headers
+    def __init__(
+        self,
+        stream: Optional[IO[bytes]] = ...,
+        filename: Union[None, Text, bytes] = ...,
+        name: Optional[Text] = ...,
+        content_type: Optional[Text] = ...,
+        content_length: Optional[int] = ...,
```

Comment:
In theory, this could be anything, since this just calls `str(content_length)` if `content_length` is not `None`. Nevertheless I'd assume this to be a type error in the vast majority of cases. Also, this mirrors the type of the `content_length` property.